devolving_music/models/song.py

```diff
@@ -7,7 +7,7 @@ class Song(models.Model):
     title = models.CharField(max_length=128)
     artists = models.ManyToManyField(Artist)
     platform_id = models.CharField(max_length=32, unique=True)
-
+   
```

Comment:
What's up with the spacing changes in this file?

Reply:
Let's revert these spacing changes since they add noise to the diff and violate PEP 8.

Reply:
Pylint says there's a trailing white space I was trying to get rid of it